extensions/amazon-lambda/runtime/pom.xml

```diff
@@ -35,6 +35,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jackson</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx</artifactId>
+        </dependency>
```

Comment:
Why the new dep?

Reply:
We exclude Vert.x from `quarkus-http`, so we need to have access to it.

Reply:
OK.

Reply:
Don't we need to add the deployment dependency in the deployment module?